src/pages/forms/createAccount.js

```diff
@@ -0,0 +1,321 @@
+import React, { useEffect, useRef, useState } from "react";
+import styles from "./createAccount.module.css";
+import { Link } from "react-router-dom";
+// import UsePasswordToggle from "../customHooks/usePasswordToggle";
+// import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useParams } from "react-router-dom";
+const CreateAccount = () => {
+  const inputData1 = useRef();
+  const inputData2 = useRef();
+  // const inputData3 = useRef();
+  // const inputData4 = useRef();
+  const inputData5 = useRef();
+  const inputData6 = useRef();
+  const inputData7 = useRef();
+  // const [PasswordInputType, ToggleIcon] = UsePasswordToggle();
+
+  // console.log(PasswordInputType, ToggleIcon);
+
+  const [showEye, setShowEye] = useState(true);
+  const [showPass, setShowPass] = useState(true);
+  const [radioMarked, setRadioMarked] = useState(false);
+  const [randomVal1, setRandomVal1] = useState(true);
+  const [randomVal2, setRandomVal2] = useState(true);
+  const [emailVal, setEmailVal] = useState(false);
+  const [passVal, setPassVal] = useState();
+  // const [ag, setAg] = useState(false);
+  // const [jj, setJj] = useState(false);
+
+  // const [ol, setOl] = useState(false);
+  // const [bz, setBz] = useState(false);
+  const [firstNameVal, setFirstNameVal] = useState(false);
+  const [lastNameVal, setLastNameVal] = useState(false);
+  const [bussNameVal, setBussNameVal] = useState(false);
+  const params = useParams();
+
+  useEffect(() => {
+    if (params.id === "brand") {
+      setRadioMarked(true);
+      // setOl(true);
+    } else {
+      setRadioMarked(false);
+      // setBz(true);
+    }
+  }, []);
+
+  const onHil = () => {
+    setRadioMarked(true);
+    // setOl(true);
+    // setBz(false);
+    setRandomVal1(true);
```

Comment:
function naming should be proper . what is setRandom1 and 2

Reply:
random1 and 2 are random values just used in place of true and false